packages/config/config/devices/0x019b/z-dim2.json

```diff
@@ -0,0 +1,255 @@
+{
+	"manufacturer": "Heatit",
+	"manufacturerId": "0x019b",
+	"label": "Z-DIM2",
+	"description": "Heatit Z-Dim2",
+	"devices": [
+		{
+			"productType": "0x0004",
+			"productId": "0x2201",
+			"zwaveAllianceId": 4666
+		},
+		{
+			"productType": "0x0002",
+			"productId": "0x2201"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic Set",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Switch Multilevel",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Power Restore Level",
+			"description": "Range between 0(off) and 99(max)",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 100,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Return to last level",
+					"value": 100
+				}
+			]
```

Comment:
For consistency, please define this like here:
https://github.com/zwave-js/node-zwave-js/blob/08d01412047db027a64efbdd90521bdcf89dbc86/packages/config/config/devices/0x031e/lzw31-sn.json#L155-L170